named-addresses/mainnet_emerald.json

```diff
@@ -0,0 +1,127 @@
+[
+    {
+        "Name": "Common Pool",
+        "Address": "oasis1qz78phkdan64g040cvqvqpwkplfqf6tj6uwcsh30",
+        "Description": "ParaTime account for rewarding token stakers."
+    },
+    {
+        "Name": "Fee Accumulator",
+        "Address": "oasis1qp3r8hgsnphajmfzfuaa8fhjag7e0yt35cjxq0u4",
+        "Description": "This account collects transaction fees of each ParaTime block that are distributed among the validators."
+    },
+    {
+        "Name": "Rewards Account",
+        "Address": "oasis1qp7x0q9qahahhjas0xde8w0v04ctp4pqzu5mhjav",
+        "Description": "ParaTime account for rewarding the compute nodes."
+    },
+    {
+        "Name": "Pending Withdrawal Account",
+        "Address": "oasis1qr677rv0dcnh7ys4yanlynysvnjtk9gnsyhvm6ln",
+        "Description": "Withdrawal from the ParaTime is performed in two steps. This account holds the tokens that are still waiting for the consensus transaction to be confirmed."
+    },
+    {
+        "Name": "Pending Delegation Account",
+        "Address": "oasis1qzcdegtf7aunxr5n5pw7n5xs3u7cmzlz9gwmq49r",
+        "Description": "Delegation from the ParaTime is performed in two steps. This account holds the tokens that are still waiting for the consensus transaction to be confirmed."
+    },
+    {
+        "Name": "Zero Address",
+        "Address": "oasis1qq2v39p9fqk997vk6742axrzqyu9v2ncyuqt8uek",
+        "Description": "This address is the oasis-standard derivation/counterpart of the Ethereum address 0x0. Sometimes used in internal data representations to indicate the absence of an account. For example, token mints are encoded as transfers from this address, and token burns as transfers to this address."
+    },
+    {
+        "Name": "Subcall Precompile",
+        "Address": "oasis1qzr543mmela3xwflqtz3e0l8jzp8tupf3v59r6qn",
+        "Description": "Interacts with Oasis Runtime SDK modules. Some transactions are performed in two steps and emit event in the next block."
+    },
+    {
+        "Name": "Gene Friend Network",
+        "Address": "oasis1qzdqfu06mgwpad0sk7p68ff5j0h9vkssk59tf9jg",
+        "Description": "ERC721"
+    },
+    {
+        "Name": "YUZUToken",
+        "Address": "oasis1qqz8706pmf38wmptl6dkcaec8yykw0rvfv7ql6fc",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "Fountain Protocol",
+        "Address": "oasis1qpssvkplnlpzdvwxpgmrhf9j5lkyvaylcvujhjhg",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "🌹RoseSwap.io",
+        "Address": "oasis1qpal7x2mn0d0jqa7wtyh7wf3klftp6aufcgzjekt",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "Wrapped ROSE",
+        "Address": "oasis1qpgcp5jzlgk4hcenaj2x82rqk8rrve2keyuc8aaf",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "YuzuSwap LP Token",
```

Comment:
Why do we have multiple "YuzuSwap LP Token" 's/"Wrapped ROSE"... here? I thought we would just have "trusted"/"official" addresses, not the list of all verified tokens. 

If we have a list of all verified contracts, can't we get that via the existing API? (https://github.com/oasisprotocol/nexus/blob/1f050d70544a0c226f70984ccf8725a84175e9ed/api/spec/v1.yaml#L1062). This seems just a duplicate of that one, with the downside that this one needs to be maintained manually.